<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PembayaranWindow</class>
 <widget class="QMainWindow" name="PembayaranWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manajemen Pembayaran</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string>QWidget { background-color: #2c3e50; }</string>
      </property>
      <layout class="QVBoxLayout" name="sidebarLayout">
       <item>
        <widget class="QLabel" name="logoLabel">
         <property name="styleSheet">
          <string>QLabel { color: white; font-size: 18px; font-weight: bold; padding: 15px; background-color: #2c3e50; }</string>
         </property>
         <property name="text">
          <string>üè®HOTEL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dashboardBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reservasiBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Reservasi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kamarBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Kamar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pelangganBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Pelanggan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pembayaranBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #34495e; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="laporanBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Laporan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logoutBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #e74c3c; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #c0392b; }</string>
         </property>
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentArea" native="true">
      <layout class="QVBoxLayout" name="contentLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="styleSheet">
          <string>QLabel { font-size: 20px; font-weight: bold; color: #2c3e50; margin-left: 20px; padding-top: 10px; }</string>
         </property>
         <property name="text">
          <string>Data Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="statsLayout">
         <item>
          <widget class="QGroupBox" name="stat1">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 229);</string>
           </property>
           <property name="title">
            <string>Berhasil</string>
           </property>
           <layout class="QVBoxLayout" name="stat1Layout">
            <item>
             <widget class="QLabel" name="stat1Value">
              <property name="styleSheet">
               <string>QLabel { color: #2ecc71; font-size: 18px; font-weight: bold; }</string>
              </property>
              <property name="text">
               <string>Rp 3.05 M</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="stat2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(215, 245, 255);</string>
           </property>
           <property name="title">
            <string>Menunggu</string>
           </property>
           <layout class="QVBoxLayout" name="stat2Layout">
            <item>
             <widget class="QLabel" name="stat2Value">
              <property name="styleSheet">
               <string>QLabel { color: #f39c12; font-size: 18px; font-weight: bold; }</string>
              </property>
              <property name="text">
               <string>Rp 750 K</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="stat3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(194, 211, 255);</string>
           </property>
           <property name="title">
            <string>Gagal</string>
           </property>
           <layout class="QVBoxLayout" name="stat3Layout">
            <item>
             <widget class="QLabel" name="stat3Value">
              <property name="styleSheet">
               <string>QLabel { color: #e74c3c; font-size: 18px; font-weight: bold; }</string>
              </property>
              <property name="text">
               <string>Rp 3 M</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="stat4">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(243, 220, 255)</string>
           </property>
           <property name="title">
            <string>Total Pendapatan</string>
           </property>
           <layout class="QVBoxLayout" name="stat4Layout">
            <item>
             <widget class="QLabel" name="stat4Value">
              <property name="styleSheet">
               <string>QLabel { color: #3498db; font-size: 18px; font-weight: bold; }</string>
              </property>
              <property name="text">
               <string>Rp 6.8 M</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QComboBox" name="statusFilter">
           <item>
            <property name="text">
             <string>Semua Status</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Berhasil</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Menunggu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gagal</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="metodeFilter">
           <item>
            <property name="text">
             <string>Semua Metode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tunai</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Transfer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QRIS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>E-Wallet</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>500</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="printBtn">
           <property name="text">
            <string>Print</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="verifiBtn">
           <property name="text">
            <string>Verifikasi</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="pembayaranTable">
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>ID Bayar</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID Res</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Jumlah</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Metode</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tgl Bayar</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Bukti</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Aksi</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>300000</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>Tunai</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>2025-10-20</string>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>Berhasil</string>
          </property>
         </item>
         <item row="0" column="6">
          <property name="text">
           <string>bukti1.jpg</string>
          </property>
         </item>
         <item row="0" column="7">
          <property name="text">
           <string>Lihat</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>1000000</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>Transfer</string>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>2025-10-21</string>
          </property>
         </item>
         <item row="1" column="5">
          <property name="text">
           <string>Berhasil</string>
          </property>
         </item>
         <item row="1" column="6">
          <property name="text">
           <string>bukti2.jpg</string>
          </property>
         </item>
         <item row="1" column="7">
          <property name="text">
           <string>Lihat</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>750000</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>QRIS</string>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>2025-10-22</string>
          </property>
         </item>
         <item row="2" column="5">
          <property name="text">
           <string>Menunggu</string>
          </property>
         </item>
         <item row="2" column="6">
          <property name="text">
           <string>bukti3.jpg</string>
          </property>
         </item>
         <item row="2" column="7">
          <property name="text">
           <string>Lihat</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>1800000</string>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string>E-Wallet</string>
          </property>
         </item>
         <item row="3" column="4">
          <property name="text">
           <string>2025-10-23</string>
          </property>
         </item>
         <item row="3" column="5">
          <property name="text">
           <string>Berhasil</string>
          </property>
         </item>
         <item row="3" column="6">
          <property name="text">
           <string>bukti4.jpg</string>
          </property>
         </item>
         <item row="3" column="7">
          <property name="text">
           <string>Lihat</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string>3000000</string>
          </property>
         </item>
         <item row="4" column="3">
          <property name="text">
           <string>Transfer</string>
          </property>
         </item>
         <item row="4" column="4">
          <property name="text">
           <string>2025-10-24</string>
          </property>
         </item>
         <item row="4" column="5">
          <property name="text">
           <string>Gagal</string>
          </property>
         </item>
         <item row="4" column="6">
          <property name="text">
           <string>bukti5.jpg</string>
          </property>
         </item>
         <item row="4" column="7">
          <property name="text">
           <string>Lihat</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalLabel">
         <property name="text">
          <string>Total Pembayaran: 5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
